src/go.mod

```diff
@@ -1,74 +1,77 @@
 module code.cloudfoundry.org/log-cache
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.0
```

Comment:
I would be very careful about this change. Not sure if there's unintended consequences or not...

Reply:
I don't think that adding the `toolchain` line is a good idea. It may result in development of this module using go1.22.0, while the release is actually built using go1.21.x

Reply:
https://tip.golang.org/doc/toolchain